portunus.csl

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="it-IT">
+  <info>
+    <title>Portunus</title>
+    <id>http://www.zotero.org/styles/portunus</id>
+    <link href="http://www.zotero.org/styles/portunus" rel="self"/>
```

Comment:
do you know the template style you used for this? Could you also link to that with rel="template"?

Reply:
I started from "Nature" template, but it's now so much different to relate it...